lib/is-electron-project.js

```diff
@@ -1,24 +0,0 @@
-var explain = require('explain-error')
-var findup = require('findup')
-var path = require('path')
-var fs = require('fs')
-
-module.exports = isElectronProject
-
-function isElectronProject (dirname, cb) {
```

Comment:
Moved this to a `hasDependency` util function because the code needed to check for typescript was exactly the same.